edwin/settings.py

```
@@ -12,38 +11,65 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-DEBUG = os.environ["DEBUG"].lower() == "true"
+DEBUG = os.getenv("DEBUG", "false").lower() == "true"
```

Non-blocking: I would error out if this env var is missing as it should be available in all envs